metaflow/plugins/__init__.py

```diff
@@ -178,6 +184,10 @@ def _merge_lists(base, overrides, attr):
     "name",
 )
 
+# Cards
```

Comment:
Strip and just put the single card for now. The import mechanism needs to be thought out a bit more IMHO so that it plays nice with metaflow_extensions. Since right now we just want to support a single card, I would just put `CARDS = [<the default card>]`